src/server/scripts/Northrend/IcecrownCitadel/boss_rotface.cpp

```diff
@@ -655,14 +656,19 @@ class spell_rotface_large_ooze_buff_combine : public SpellScriptLoader
 
             void HandleScript(SpellEffIndex /*effIndex*/)
             {
-                if (!(GetHitCreature() && GetHitCreature()->IsAlive()))
+                Unit* caster = GetCaster();
+
+                if (!(GetHitCreature() && GetHitCreature()->IsAlive() && caster))
                     return;
 
-                if (Aura* unstable = GetCaster()->GetAura(SPELL_UNSTABLE_OOZE))
+                if (Aura* unstable = caster->GetAura(SPELL_UNSTABLE_OOZE))
                 {
                     uint8 newStack = uint8(unstable->GetStackAmount()+1);
                     unstable->SetStackAmount(newStack);
 
+                    if (unstable->GetStackAmount() > 2)
+                        caster->SetAuraStack(SPELL_OOZE_GROW, caster, unstable->GetStackAmount() - 2);
```

Comment:
Shouldn't both `unstable->GetStackAmount()` be `newStack`

Reply:
newstack starts 1,
don't want spell to be executed unless an actual merge with the big ooze occurs